weave/bin/modules/config.py

```
@@ -142,4 +143,52 @@ def get_service_by_id(service_id):
+    if service_identifier in services:
+        service_info = services[service_identifier]
+        container_patterns = service_info.get("container_patterns", [])
+        
+        # Try to match container patterns to docker-compose services
+        for docker_svc in docker_services:
+            for pattern in container_patterns:
+                if pattern in docker_svc:
+                    return docker_svc
+    
+    # Check if it's a display name in our config
+    for service_id, info in services.items():
```

Ruff (0.11.9)

Loop control variable `service_id` not used within loop body

Rename unused `service_id` to `_service_id`

(B007)